token-allocation/README.md

```diff
@@ -0,0 +1,78 @@
+# Chainlink Token Allocation Price Adapter
+
+The adapter calculates the total value in the currency selected for the selected tokens
+
+## Configuration
+
+The adapter takes the following environment variables:
+
+- `DATA_PROVIDER`: Data provider to use. Some of them require an `API_KEY`(K). Options available:
+    - `amberdata` (K)
+    - `coinapi`(K)
+    - `coingecko`
+    - `coinmarketcap`(K)
+    - `coinpaprika`
+    - `cryptocompare`
+    - `kaiko` (K)
+    - `nomics`(K)
+- `API_KEY`: For those data providers who need an api key
+- `DEFAULT_CURRENCY` (Optional): Currency that the price will be fetched by default. `USD` used by default
+
+
+## Input Params
+
+- `components`: Array of the token symbols. 
+- `units`: Array of balances in wei of each token.
+- `currency` (optional). Currency we want the price on. `DEFAULT_CURRENCY` by default
```

Comment:
More of a generic question, but would it be easy to extend this adapter with a param for "endpoint"?
Say the default currently is to get the prices, could it easily be extended to say marketcap percentage?

Reply:
I don't think it would be hard, and make sense to have that functionality in this adapter.